CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.2.0] - 2020-11-23
```

Comment:
Shouldn't' this be 3.0.0?

Reply:
why?

Reply:
There are breaking changes like removed `route53_enabled`.